calico-vpp-agent/cni/network_vpp.go

```diff
@@ -319,6 +326,10 @@ func (s *Server) DelVppInterface(podSpec *storage.LocalPodSpec) {
 		}
 	}
 
+	/* RPF */
+	s.log.Infof("pod(del) RPF VRF")
+	s.DeletePodRPFVRF(podSpec)
```

Comment:
`DeactivateStrictRPF()` or `DisableStrictRPF()` ? Just to make it clear it's the counterpart of `ActivateStrictRPF()`